ServerlessLibraryAPI/ClientApp/src/components/DetailView/DetailView.js

```diff
@@ -1,58 +1,18 @@
 import React, { Component } from "react";
-import ReactMarkdown from "markdown-to-jsx";
 import { connect } from "react-redux";
 import { libraryService } from "../../services";
-import {
-  IconButton,
-  Label,
-  PrimaryButton,
-  Link,
-  Pivot,
-  PivotItem,
-  PivotLinkSize,
-  ScrollablePane
-} from "office-ui-fabric-react/lib/index";
+import ActionBar from "./ActionBar";
+import DetailPageContent from "./DetailPageContent";
+import DetailPageHeader from "./DetailPageHeader";
+
 import { sampleActions } from "../../actions/sampleActions";
-import MetricBar from "../MetricBar/MetricBar";
-import { getTheme, mergeStyleSets } from "office-ui-fabric-react";
-import "./DetailView.css";
-const theme = getTheme();
-const classNames = mergeStyleSets({
-  wrapper: {
-    minHeight: "60vh",
-    position: "relative",
-    maxHeight: "inherit"
-  },
-  pane: {
-    minHeight: "60vh",
-    border: "1px solid " + theme.palette.neutralLight,
-    whiteSpace: "pre"
-  },
-  sticky: {
-    color: theme.palette.neutralDark,
-    padding: "5px 20px 5px 10px",
-    fontSize: "13px",
-    borderTop: "1px solid " + theme.palette.black,
-    borderBottom: "1px solid " + theme.palette.black
-  },
-  textContent: {
-    padding: "15px 10px"
-  }
-});
 
 class DetailView extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      sample: {},
-      armTemplateText: "",
-      markdownText: "",
-      selectedKey: "licence"
+      sample: {}
     };
-
-    this.handleBackButtonClick = this.handleBackButtonClick.bind(this);
-    this.handleLinkClick = this.handleLinkClick.bind(this);
-    this.getVScodeLink = this.getVScodeLink.bind(this);
   }
 
   handleBackButtonClick() {
```

Comment:
At line 28: this.setState({ repository: currentItem.repository });
I am not able to see 'repository' in the state